e2e/test/provisioning/ProvisioningServiceClientE2ETests.cs

```diff
@@ -110,6 +111,45 @@ public async Task ProvisioningServiceClient_GetEnrollmentGroupAttestation_Symmet
             await ProvisioningServiceClient_GetEnrollmentGroupAttestation(AttestationMechanismType.SymmetricKey);
         }
 
+        [LoggedTestMethod]
+        public async Task ProvisioningServiceClient_TokenCredentialAuth_Success()
+        {
+            using var provisioningServiceClient = ProvisioningServiceClient.Create(
+                TestConfiguration.Provisioning.GetProvisioningHostName(),
+                TestConfiguration.Provisioning.GetClientSecretCredential());
+
+            IndividualEnrollment individualEnrollment = await CreateIndividualEnrollment(provisioningServiceClient, AttestationMechanismType.SymmetricKey, null, AllocationPolicy.Static, null, null, null);
```

Comment:
When lines get this long, we like to break them up. For this one, parameters can each get their own line. Also, in product code and tests (not samples) we append `.ConfigureAwait(false)` to async calls.

```suggestion
            IndividualEnrollment individualEnrollment = await CreateIndividualEnrollment(
                    provisioningServiceClient,
                    AttestationMechanismType.SymmetricKey,
                    null,
                    AllocationPolicy.Static,
                    null,
                    null,
                    null)
                .ConfigureAwait(false);
```